dss/api/files.py

```diff
@@ -329,5 +328,7 @@ class CopyMode(Enum):
                 f"file with UUID {uuid} and version {version} already exists")
         status_code = requests.codes.ok
 
-    return jsonify(
-        dict(version=version)), status_code
+    if should_cache_file(content_type=content_type, size=size):
+        start_file_checkout(replica=replica, blob_key=dst_key)
```

Comment:
Does `start_file_checkout` react appropriately when the copy is asynchronous (CopyMode.COPY_ASYNC above)?

Reply:
I've made the (temporary) check to account for this.